docs/articles/ambassador_pattern_linking.md

```diff
@@ -81,42 +81,43 @@ On the Docker host (192.168.1.52) that Redis will run on:
     ^D
 
     # add redis ambassador
-    $ docker run -t -i --link redis:redis --name redis_ambassador -p 6379:6379 busybox sh
+    $ docker run -t -i --link redis:redis --name redis_ambassador -p 6379:6379 alpine:3.2 sh
 
 In the `redis_ambassador` container, you can see the linked Redis
 containers `env`:
 
-    $ env
+    / # env
```

Comment:
Sorry, I'm not sure what you mean! Makes sense to revert it or to leave it as the new prompt?